pkg/machine/ignition/ignition.go

```diff
@@ -879,3 +912,17 @@ func DefaultReadyUnitFile() parser.UnitFile {
 	u.Add("Install", "RequiredBy", "default.target")
 	return *u
 }
+
+func GetRosettaActivationUnitFile() *parser.UnitFile {
+    rosettaUnit := parser.NewUnitFile()
```

Comment:
Are you intentionally not making use of `systemd-binfmt`? See https://github.com/crc-org/vfkit/blob/main/doc/usage.md#description-12

Reply:
As I replied to your comment on `update.go`, I created another service (rosetta-activation.service) that can be controlled to avoid systemd errors.

Reply:
@cfergeau I explained in my latest comment.
https://github.com/containers/podman/pull/21670#issuecomment-1975002363